shell/platform/common/cpp/client_wrapper/basic_message_channel_unittests.cc

```diff
@@ -0,0 +1,93 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/basic_message_channel.h"
+
+#include <memory>
+#include <string>
+
+#include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/binary_messenger.h"
+#include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/standard_message_codec.h"
+#include "gtest/gtest.h"
+
+namespace flutter {
+
+namespace {
+
+class TestBinaryMessenger : public BinaryMessenger {
```

Comment:
Would `TestBinaryMessenger` be better named as `MockBinaryMessenger`, and use the `Test` prefix for the subjects being tests?

Reply:
Since it's manual, Fake might be better than Mock?

I'm going to land as-is for now, since at least the registrar unit test already uses this name, and we can revisit naming for everything at once later.